pom.xml

```diff
@@ -119,6 +121,7 @@
                         <environmentVariables>
                             <ICEDTEA_WEB_SPLASH>DEFAULT</ICEDTEA_WEB_SPLASH>
                         </environmentVariables>
+                        <redirectTestOutputToFile>true</redirectTestOutputToFile>
```

Comment:
The output is now much cleaner.
Problem is, in travis we can no longer see why a test may have failed because all output is gone

Reply:
Does this project own an S3 bucket? This way we could upload the text files as artifacts

Reply:
At the moment we have no good way to store the output. Please remove that line so that the PR can be merged :)

Reply:
Done! :)